### DISCUSSION TITLE
blogs/230315-install-on-linux/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1603] [Announcements]
# blogs/230315-install-on-linux/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/blogs/230315-install-on-linux/

**Comment by Adityayxt** [comment]
按照这个教程操作，用了半小时安装成功了
